README.md

```diff
@@ -302,6 +303,28 @@ would not: in all classes of browsers, it would fetch the polyfill unconditional
 
 which will work as desired in all classes of browser.
 
+#### Alternating logic based on the presence of a built-in module
+
+_See further discussion of this case in the issue tracker: [#61](https://github.com/domenic/package-name-maps/issues/61)._
+
+Not all fallbacks take the role of running one piece of code. For example, sometimes, one code path is to be taken if a particular platform API exists, and another code path is taken if it doesn't exist. The import maps proposal does not aim to solve all such scenarios in a built-in way; instead, The Stage 2 TC39 proposal [top-level await](https://github.com/tc39/proposal-top-level-await) can be used to meet some of these use cases. Note that top-level await blocks import of the module containing the await.
```

Comment:
It doesn't block import; it suspends execution like a normal `await` does. So I'll remove this last sentence as it seems to add confusion.